scripts/buildtools/install_buildtools.sh

```diff
@@ -36,9 +72,14 @@ function install_go_module {
     fi
 }
 
-install_go_module golang.org/x/lint golang.org/x/lint/golint
-install_go_module golang.org/x/tools golang.org/x/tools/cmd/stringer
-install_go_module github.com/go-swagger/go-swagger github.com/go-swagger/go-swagger/cmd/swagger
-install_go_module github.com/algorand/msgp
+if [[ "${BUILDTOOLS_INSTALL}" == "ALL" ]]; then
```

Comment:
instead of testing it here, move the test to be inside `install_go_module` -
this would ensure that the only installed components by this script are such that are explicitly listed here.

Reply:
I changed this suggestion slightly to require that packages have a version specified. I think that should accomplish the same goal.

Reply:
I don't think that this change make this into a requirement. you could call
```bash
install_buildtools.sh GitHub.com/malware/go-malware
```
which would install the above. My intention was to limit the components to one of the ones we explicitly listed.

Reply:
That would fail because it has no version listed in the buildtools `go.mod`